### PR TITLE
Package also circuits

### PR DESCRIPTION
**Pull request by acomodi**
This PR is to add also the selected tests' circuits to the install directory.

It is possible to add a new test to the installed ones through the CMake option `INSTALL_CIRCUIT` when calling the `add_fpga_target` function.

### CODE REVIEW

File: common/cmake/devices.cmake [hunk 1]
```diff
@@ -1447,16 +1456,40 @@ function(ADD_FPGA_TARGET)
...
 
   if(NOT "${ADD_FPGA_TARGET_INPUT_SDC_FILE}" STREQUAL "")
     append_file_dependency(VPR_DEPS ${ADD_FPGA_TARGET_INPUT_SDC_FILE})
     get_file_location(SDC_LOCATION ${ADD_FPGA_TARGET_INPUT_SDC_FILE})
     set(SDC_ARG --sdc_file ${SDC_LOCATION})
+    set(SDC_FILE ${SDC_LOCATION})
+  endif()
+
+  if (${ADD_FPGA_TARGET_INSTALL_CIRCUIT})
+    add_custom_target(
```
Comment (litghost) [inline common/cmake/devices.cmake:1475]: Not sure what this custom target is for?

File: .github/kokoro/continuous-install-200t.cfg [hunk 2]
```diff
@@ -11,7 +11,7 @@ timeout_mins: 4320
 action {
   define_artifacts {
     # File types
-    regex: "**/symbiflow-arch-defs-install-200t*.tar.xz"
+    regex: "**/symbiflow-arch-defs-*.tar.xz"
```
Comment (litghost) [inline .github/kokoro/continuous-install-200t.cfg:14]: Don't see a change to the template python script?

**Review by HackerFoo — APPROVED** [review]

**Review by mithro — APPROVED** [review]
LGTM if you fix @litghost's stuff.

File: .github/kokoro/continuous-install-200t.cfg (hunk 2 above)
Comment (acomodi) [reply .github/kokoro/continuous-install-200t.cfg:14]: Done.

I am thinking whether we need the install-200t CI as the 200T is now built and packaged in the install one.

File: common/cmake/devices.cmake (hunk 1 above)
Comment (acomodi) [reply common/cmake/devices.cmake]: It was to build the dependency to install the eblif, but I think it is not needed actually. Removed

Comment (acomodi) [reply common/cmake/devices.cmake:1475]: Actually this target is needed when calling the install target, otherwise the eblifs do not get built.

**Comment by acomodi** [comment]
Install targets fail with no log output: [Link to failed build](https://source.cloud.google.com/results/invocations/e378b652-7596-4d03-b638-01bc66627d24/log).

This seems similar to the problems had when packaging the 200T alongside with the other devices: https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1747#issuecomment-721100912

**Comment by litghost** [comment]
> Install targets fail with no log output: [Link to failed build](https://source.cloud.google.com/results/invocations/e378b652-7596-4d03-b638-01bc66627d24/log).

```
CMake Error at xc/xc7/tests/soc/ibex/cmake_install.cmake:50 (file):
  file INSTALL cannot find
  "/tmpfs/src/github/symbiflow-arch-defs-presubmit-install/build/xc/xc7/tests/soc/ibex/pins_artya7.sdc":
  No such file or directory.
Call Stack (most recent call first):
  xc/xc7/tests/soc/cmake_install.cmake:52 (include)
  xc/xc7/tests/cmake_install.cmake:177 (include)
  xc/xc7/cmake_install.cmake:88 (include)
  xc/cmake_install.cmake:52 (include)
  cmake_install.cmake:72 (include)
```

**Comment by acomodi** [comment]
@litghost All right, thanks. Something wrong in the PR then, I'll fix that

**Pull request merged**